pkg/controller/plan/migration.go

```diff
@@ -1095,12 +1095,16 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 		}
 		step.MarkStarted()
 		step.Phase = Running
-		err = r.ensureGuestConversionPod(vm)
+		ready, err := r.ensureGuestConversionPod(vm)
 		if err != nil {
 			step.AddError(err.Error())
 			err = nil
 			break
 		}
+		if !ready {
+			r.Log.Info("virt-v2v pod isn't ready yet")
+			return nil
```

Comment:
```suggestion
			return
```

Reply:
the error var is not in the same scope here so we need to return nil manually.